packages/rpc-core/src/rpc-methods/getFeeForMessage.ts

```diff
@@ -0,0 +1,18 @@
+import { Commitment, RpcResponse, Slot, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+/** Fee corresponding to the message at the specified blockhash */
+type GetFeeForMessageApiResponse = RpcResponse<U64UnsafeBeyond2Pow53Minus1 | null>;
+
+export interface GetFeeForMessageApi {
+    /**
+     * Returns the fee the network will charge for a particular Message
+     */
+    getFeeForMessage(
+        /** Base-64 encoded message */
+        message: string, // TODO: May not be string
```

Comment:
`Base64EncodedBytes`?

Reply:
I think that makes sense, is that de facto for tx message?

Reply:
I want to say yes but there's probably somewhere we use some other encoding for some reason 😅 In this case though the RPC docs are clear that it's base64 and that's exactly what this type is for!

Reply:
Got it, going with that then!